registry/configuring-registry-operator.adoc

```diff
@@ -11,7 +11,7 @@ registry storage.
 [NOTE]
 ====
 Storage is only automatically configured when you install an
-installer-provisioned infrastructure cluster on AWS, GCP, or Azure.
```

Comment:
@adambkaplan - I updated this note to include OpenStack. Is that accurate?

Reply:
@bmcelvee correct - we support IPI on OpenStack.

Reply:
Thanks!